app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User'
```

Comment:
* Kindly Create model classes for all entities and Set up associations between models as shown in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). [Refer to project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md#:~:text=Create%20model%20classes,associations%20between%20models.)  foreign keys should be `author_id` and not `user_id` as shown above ⬆️